e2e/app.po.ts

```diff
@@ -195,6 +197,16 @@ export class AppPage {
     }).then(jsonString => JSON.parse(<string> jsonString));
   }
 
+  waitForElementToBePresent(element) {
+    browser.wait(function () {
+        return element.isPresent();
+    }, 60000);
+
+    browser.wait(function () {
+        return element.isDisplayed();
+    }, 60000);
+  }
+
```

Comment:
I like the idea to wait for element in generatl. Maybe we could wrap this with `ExpectedConditions` and use either [`elementToBeClicable()`](https://github.com/angular/protractor/blob/master/lib/expectedConditions.ts#L188) or [`visibilityOf`](https://github.com/angular/protractor/blob/master/lib/expectedConditions.ts#L393). Wdyt?
Cc @jludvice